content/04.results.md

```diff
@@ -3,11 +3,62 @@
+The discriminative model over takes the generative model as more label functions are incorporated. 
+This similar trend is seen for the Compound treats Disease (CtD) edge type; however, both the generative and discriminative model perform equally instead of one outcompeting another.
+For the Compound binds Gene (CbG) edge type, the discriminative model consistently underperforms the generative model.
+Lastly for the Gene interacts Gene (GiG) edge type the discriminative model starts off underperforming the generative model.
+Just like with CtD and DaG performance improves as more label functions are added.  
+Looking at the off-diagonals in figures {@fig:discriminative_model_auroc} and {@fig:discriminative_model_auprc}, the discriminative model shows variable performance depending on the edge type.
+For the DaG row, the discriminative model consistently underperforms the generative model, that is trained using label functions from other edge types. 
+The exception towards this trend is when label functions from all edge types are used.
+The inverse effect can be seen for CtD row.
+The discriminative model starts off out performing the generative model, but then shifts into underperformance as more label functions are added.
+For both the CbG and GiG rows performance starts off with an increasing trend, but then either plateaus or sharply declines.
+A perfect example of this the described trend can be seen in the last two rows and first two columns.
+When one label function is added, performance spikes and then sharply declines as more label functions are added.
+Interestingly, the discriminative model does well in predicting the CbG edge type, when using the generative model trained on GiG label functions.
+This trend doesn't hold for the reverse as the discriminative model slightly underperforms the generative model.
+Lastly for CbG and GiG using all label functions didn't aid in the discriminative model performance.
 
-## Random Noise Generative Model
-place the results of random label function experiment
+![
+Grid of Area Under the Receiver Operating Curve (AUROC) scores for each discriminative model trained using generated labels from the generative models.
+The rows depict the relationship each model is trying to predict and the columns are the relationship specific sources each label function was sampled from. 
+For example, the top-left most square depicts the discriminator model predicting Disease associates Gene (DaG) sentences, while randomly sampling label functions designed to predict the DaG relationship. 
+The square towards the right depicts the discriminative model predicting DaG sentences, while randomly sampling label functions designed to predict the Compound treats Disease (CtD) relationship.
+This pattern continues filling out the rest of the grid.
+The last most column consists of pooling every relationship specific label function and proceeding as above.
```

Comment:
Put in what the error bars represent. You should do this in all figures with error bars.